client/me/purchases/controller.jsx

```diff
@@ -19,44 +19,60 @@ import EditCardDetails from './payment/edit-card-details';
 import Main from 'components/main';
 import ManagePurchase from './manage-purchase';
 import NoSitesMessage from 'components/empty-content/no-sites-message';
-import * as paths from './paths';
 import PurchasesHeader from './purchases-list/header';
 import PurchasesList from './purchases-list';
-import { concatTitle, recordPageView } from 'lib/react-helpers';
+import { concatTitle } from 'lib/react-helpers';
 import { setDocumentHeadTitle } from 'state/document-head/actions';
 import titles from './titles';
 import userFactory from 'lib/user';
 import { makeLayout, render as clientRender } from 'controller';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
 
-const recordPurchasesPageView = partial( recordPageView, partial.placeholder, 'Purchases' );
 const user = userFactory();
 
 // FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
 function setTitle( context, ...title ) {
 	context.store.dispatch( setDocumentHeadTitle( concatTitle( titles.purchases, ...title ) ) );
 }
 
+const userHasNoSites = user.get().site_count <= 0;
```

Comment:
This feels dangerous to call this at the module level, can we move this inside to where this is being called in the controller?

Reply:
alternatively make this a thunk ;)
```
const userHasNoSites = () => user.get().site_count <= 0;
```